src/components/modal/modal.spec.js

```diff
@@ -202,4 +158,50 @@ describe('Modal', () => {
       });
     });
   });
+
+  describe('when the modal is open', () => {
+    let domNode;
+    let escapeKeyEvent;
+    const onCancelFn = jest.fn();
+
+    beforeEach(() => {
```

Comment:
shouldn't we assign `jest.fn()` in a beforeEeach fn? For now we create one for all tests

Reply:
there is no need for that if you do a `mockReset`